pkgs/applications/networking/cluster/terraform-providers/generic.nix

```diff
@@ -0,0 +1,13 @@
+{ buildGoPackage, src }:
+# A specialized buildGoPackage for the terraform provider plugins
+{ pname, version, ... }@attrs:
+buildGoPackage ({
+  name = "${pname}-${version}";
```

Comment:
what if you pass in `name` instead of `pname` and then access the inner name via `attrs.name`?

```nix
{ name, version, ... }@attrs:
buildGoPackage ({
  name = "${attrs.name}-${version}"
```